db/mysql/schema.sql

```diff
@@ -438,6 +438,8 @@ ADD FOREIGN KEY (`user_id`)
     REFERENCES `usr_user` (`id`)
 	ON UPDATE CASCADE ON DELETE CASCADE;
 
+ALTER TABLE ppl_model ADD UNIQUE (name(255));
```

Comment:
The problem is, that MySQL does not support such big indexes. We will either limit uniqueness on smaller size, or we will have to use innodb_large_prefix options. (http://mechanics.flite.com/blog/2014/07/29/using-innodb-large-prefix-to-avoid-error-1071/)